index.php

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title alt='page-title-collected-cats'>Collected Cats</title>
+    <link alt='cat-icon' rel="icon" type="image/png?" href="icon/cat.png"/>
+
+    <link href="winter-collector-page.css" rel="stylesheet">
+    <link href="normalize.css" rel="stylesheet">
+</head>
+<body>
+
+
+<nav class="navBar" alt="navigation-bar">
+    <ul class="navBarItems" alt="navigation-bar-links">
+        <li class="navBarItem"><a class="navBarLink" href="#" alt="home">Home</a></li>
+        <li class="navBarItem"><a class="navBarLink" href="#" alt="your-profile">Your Profile</a></li>
+        <li class="navBarItem"><a class="navBarLink" href="#" alt="add-new-cats">Add New Cats?</a></li>
+        <li class="navBarItem"><a class="navBarLink" href="#" alt="contact">Contact</a></li>
+    </ul>
+</nav>
+
+<h1>Collected Cats</h1>
+
+<h2>Welcome to your cat collection! <br/>Here you can see a list of all your cats...</h2>
```

Comment:
sorry I never user <br> but in here you put it <br/> just wondered if the / is right... 
You don't need to change it if its working just wondered. (I don't use it because Rayna once told me we shouldn't use it... but is easier with it!! 👍 )

Reply:
i think `<br>` tag is okay here because it's separating only the text and not the page elements. doing this in another way would affect other aspects of the page